packages/signature-v4-multi-region/src/load-crt.node.ts

```diff
@@ -0,0 +1,20 @@
+import { signatureV4CrtContainer } from "./signature-v4-crt-container";
```

Comment:
We don't explicitly add `.node` suffix.

Rename this to `load-crt.ts`?
The change needs to be done in package.json overrides too.